docs/content/usage/functions.en.md

```diff
@@ -38,6 +38,8 @@ $ fission route create --function hello --url /hello
 trigger '5327e9a7-6d87-4533-a4fb-c67f55b1e492' created
 ```
 
+Setup [FISSION_ROUTER](https://docs.fission.io/installation/env_vars/) environment variable if you haven't already.
```

Comment:
You can use `[FISSION_ROUTER](../installation/env_vars/)` instead

Reply:
Done. Thank you :)

Reply:
Sorry, my bad. I found it should be `../../installation/env_vars/`. After this change, we are good to go.

Reply:
Haha, no problem. I will fix it.